src/CitMovie.Data/JobCategoryRepository/JobCategoryRepository.cs

```diff
@@ -0,0 +1,33 @@
+using CitMovie.Models.Dto;
+using Microsoft.EntityFrameworkCore;
+
+namespace CitMovie.Data.JobCategoryRepository
+{
+    public class JobCategoryRepository : IJobCategoryRepository
+    {
+        private readonly DataContext _context;
+
+        public JobCategoryRepository(DataContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IList<JobCategoryDto>> GetAllJobCategoriesAsync(int page, int pageSize)
```

Comment:
To my understanding here we return the Domain object and not dto.  In the manager we do the conversion